lib/src/components/NxDropdown/stateful/__tests__/NxStatefulDropdown.test.tsx

```diff
@@ -5,192 +5,132 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { act } from 'react-dom/test-utils';
-import { mount, ReactWrapper } from 'enzyme';
-import 'jest-enzyme';
-import * as enzymeUtils from '../../../../__testutils__/enzymeUtils';
 
-import NxButton from '../../../NxButton/NxButton';
-import NxDropdown from '../../NxDropdown';
+import { act, render, screen, fireEvent } from '@testing-library/react';
+import { rtlRender, userEvent } from '../../../../__testutils__/rtlUtils';
+
 import NxStatefulDropdown, { Props } from '../NxStatefulDropdown';
 
 describe('NxStatefulDropdown', () => {
```

Comment:
Since RTL can't test that `NxStatefulDropdown` depends on `NxDropdown`, these tests shouldn't assume that `NxStatefulDropdown` also implements all of the common behavior that it inherits from `NxDropdown`.  That is, ALL of `NxStatefulDropdown`'s behaviors must be tested in this file, including the behaviors that duplicate what's tested on `NxDropdown` directly.  This is how it is for our stateful components in general - their test files must duplicate a large amount of the tests from the corresponding stateless component.

Reply:
> Since RTL can't test that `NxStatefulDropdown` depends on `NxDropdown`, these tests shouldn't assume that `NxStatefulDropdown` also implements all of the common behavior that it inherits from `NxDropdown`. That is, ALL of `NxStatefulDropdown`'s behaviors must be tested in this file, including the behaviors that duplicate what's tested on `NxDropdown` directly. This is how it is for our stateful components in general - their test files must duplicate a large amount of the tests from the corresponding stateless component.

Updated stateful test here: 6173c18d188592b3141d2c02f7f670e4d01b4715